examples/customhandler/handler.go

```diff
@@ -2,11 +2,11 @@ package main
 
 import (
 	"context"
+	transactionhandler "github.com/bitcoin-sv/arc/pkg/api/transaction_handler"
```

Comment:
No need for alias as rule has been excluded

Reply:
Removed